src/types/EnumerationMemberType.ts

```diff
@@ -0,0 +1,30 @@
+import { Element } from 'libxmljs';
+
+import ns from '../util/ns';
+import { Status } from '../enum';
+import * as Parsers from '../model/parsers';
+
+export default class EnumerationMemberType {
+  public description: string;
```

Comment:
These should be read only to prevent outside modification.

Reply:
We should do this in a bunch of places in here at some point... will file it as an enhancement

Reply:
Filed: https://github.com/128technology/yinz/issues/26